src/parsers/case.cr

```diff
@@ -13,7 +13,7 @@ module Mint
 
         whitespace
 
-        char '(', CaseExpectedOpeningParentheses
```

Comment:
`CaseExpectedOpeningParentheses` can be removed, right?

Reply:
Yep :+1: